ullyses/wrapper.py

```diff
@@ -30,6 +30,12 @@ class Ullyses_SegmentList(SegmentList):
 
     """
 
+    # Segments that should be ignored for each cenwave.
```

Comment:
As written, when you make a new Ullyses_Segmentlist (or anything that derives from Ullyses_SegmentList), you will make any other instances that derive from SegmentList have the same bad_segments, even if they're not Ullyses_SegmentLists.  It probably won't make any difference to Ullyses, since all the class instances you make are or derive from Ullyses_SegmentList, but it could possibly cause unexpected side effects.  Using self.bad_segments limits the effect to the classes you create this way

Reply:
Can you help me figure out how do this? `self` isn't yet defined in `Ullyses_SegmentList` so I tried adding an init, but it's not actually passing `self.bad_segments` into `SegmentList` and using it. Here's what I tried:
```
class Ullyses_SegmentList(SegmentList):
    """This class is a mixin to add the project-specific write function and functions
    to populate the target name and coordinates

    """

    # Segments that should be ignored for each cenwave.
    # For ULLYSES this is G230L/2635/NUVC and G230L/2950/NUVC
    def __init__(self, instrument, grating, path='.'):
        self.bad_segments = {2635: 'NUVC', 2950: 'NUVC'}
        SegmentList.__init__(self, instrument, grating, path)
```

I put a print statement in coadd to see if it was actually using the bad_segments:
```
                                if hasattr(self, "bad_segments"):
                                    if cenwave in self.bad_segments and self.bad_segments[cenwave] == row['SEGMENT']:
                                        print(f"Skipped bad segment {cenwave}/{row['SEGMENT']}")
                                        continue
```